File: Go/ConsolePrint/src.go
package main

import (
	"fmt"
	"log"
	"time"
)

func main() {
	start := time.Now()

	ti := 5000
	tj := 20
	var p = ti * tj
	var s int
	for i := 0; i < ti; i++ {
		for j := 0; j < tj; j++ {
			var r = s * 100 / p
			fmt.Println(r)
		}
		s = s + tj
	}

	elapsed := time.Since(start)
	log.Printf("Execution time: %s", elapsed)
	fmt.Scanf("h")
}

File: Rust/ConsolePrint/src.rs
use std::io;

fn main() {
    use std::time::Instant;
    let now = Instant::now();

    let ti: u32 = 5000;
    let tj: u32 = 20;
    let p = ti * tj;
    let mut s: u32 = 0;
    for i in 0..ti {
        for j in 0..tj {
            println!("{}", s * 100 / p);
        }
        s = s + tj;
    }
    let elapsed = now.elapsed();
    println!("Execution time: {:.2?}", elapsed);

    io::stdin().read_line(&mut String::new()).unwrap();
}
